apps/cookbook/src/app/examples/header-example/examples/combined.ts

```diff
@@ -1,10 +1,13 @@
 import { Component } from '@angular/core';
+import { ToastConfig, ToastController } from '@kirbydesign/designsystem';
 
 import { config as customSectionConfig } from './custom-section';
 
 const config = {
   selector: 'cookbook-header-example-combined',
-  template: `<kirby-header [title]="'Title'" value="12.345,67" valueUnit="USD" subtitle1="Subtitle one" subtitle2="Subtitle two">
+  template: `<kirby-header [title]="'Title'" value="12.345,67" valueUnit="USD" subtitle1="Subtitle one" subtitle2="Subtitle two" (titleClick)="onTitleClick($event)">
+  <kirby-icon name="arrow-down" *kirbyHeaderTitleActionIcon></kirby-icon>
```

Comment:
For some reason I am not seeing this reflected in the combined example in the cookbook. There is no clickable title or arrow icon 🤔 Or am I looking at the wrong thing? 

<img width="1046" alt="image" src="https://user-images.githubusercontent.com/42470636/236160334-26ba2b43-2911-4773-8566-0229586202db.png">

Reply:
Ah wait, this branch probably just needs to be synced with develop for this to make sense.

Reply:
@RasmusKjeldgaard I see your point, but we only wrap examples in the iPhone frame, when the example is related to Page, i.e. we need to see how it scrolls, fades into the toolbar etc. This example doesn't showcase that.
And on another note, it's not different from the Toast example itself, that isn't wrapped either: https://cookbook.kirby.design/#/home/showcase/toast